src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,18 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private static final int AMOUNT_MODELS = 3;
+    private List<Bulldozer> listOfBuldozers = new ArrayList<>();
```

Comment:
There is a typo in the variable name `listOfBuldozers`. It should be `listOfBulldozers` to match the correct spelling of 'Bulldozer'.